sample-app/src/main/java/com/sinch/sample/voice/conferences/KickParticipant.java

```diff
@@ -0,0 +1,29 @@
+package com.sinch.sample.voice.conferences;
+
+import com.sinch.sample.BaseApplication;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+public class KickParticipant extends BaseApplication {
+
+  private static final Logger LOGGER = Logger.getLogger(KickParticipant.class.getName());
+
+  public KickParticipant() throws IOException {}
+
+  public static void main(String[] args) {
+    try {
+      new KickParticipant().run();
+    } catch (Exception e) {
+      LOGGER.severe(e.getMessage());
+      e.printStackTrace();
+    }
+  }
+
+  public void run() {
+
+    String callId = "e429ecbb-6057-4756-8148-2b498f968136";
```

Comment:
This data should come from a config file, right?

Reply:
Right, it could but because of very volatile I did not handle it from config
I'll do for consistency (will change it with incoming work related to Voice API)